lib/travis/build/script/dart.rb

```diff
@@ -92,13 +92,26 @@ def setup
           sh.export 'PUB_ENVIRONMENT', 'travis'
 
           sh.fold 'dart_install' do
-            sh.echo 'Installing Dart', ansi: :yellow
+            # Install SDK and set environment variables.
+            sh.echo "Installing Dart on #{os}", ansi: :yellow
             sh.cmd "curl --connect-timeout 15 --retry 5 #{archive_url}/sdk/dartsdk-#{os}-x64-release.zip > ${TRAVIS_HOME}/dartsdk.zip"
             sh.cmd "unzip ${TRAVIS_HOME}/dartsdk.zip -d ${TRAVIS_HOME} > /dev/null"
             sh.cmd "rm ${TRAVIS_HOME}/dartsdk.zip"
             sh.cmd 'export DART_SDK="${TRAVIS_HOME}/dart-sdk"'
             sh.cmd 'export PATH="$DART_SDK/bin:$PATH"'
             sh.cmd 'export PATH="${TRAVIS_HOME}/.pub-cache/bin:$PATH"'
+
+            if os == 'windows'
+              # Define commands; on Windows git bash requires .bat extensions
+              # https://github.com/msysgit/msysgit/issues/101
+              sh.raw 'function dart2js() { dart2js.bat "$@"; }'
+              sh.raw 'function dartanalyzer() { dartanalyzer.bat "$@"; }'
+              sh.raw 'function dartdevc() { dartdevc.bat "$@"; }'
+              sh.raw 'function dartdevk() { dartdevk.bat "$@"; }'
+              sh.raw 'function dartdoc() { dartdoc.bat "$@"; }'
+              sh.raw 'function dartfmt() { dartfmt.bat "$@"; }'
+              sh.raw 'function pub() { pub.bat "$@"; }'
```

Comment:
Is `dart` working differently? Should a `dart` command be also defined?

Reply:
I think the issue is only with `.bat` files inside the Bash shell on Windows. I think `.exe`'s are selected correctly in preference to the extensionless files.